plugins/available/plenv.plugin.bash

```diff
@@ -0,0 +1,17 @@
+# plugin for plenv
+
+cite about-plugin
+about-plugin 'plenv plugin for Perl'
+
+pathmunge "${HOME}/.plenv/bin"
+
+if [[ `which plenv` ]] ; then
+
+  # init plenv
+  eval "$(plenv init -)"
+
+  # Load the auto-completion script if it exists.
+  [[ -e ~/.plenv/completions/plenv.bash ]] && source ~/.plenv/completions/plenv.bash
+
+else echo "Unable to find plenv. See https://github.com/tokuhirom/plenv for installation"
```

Comment:
Not sure whether this is a good idea. We have had several reports where printing messages like this has caused issues when e.g. using Bash-it over SSH. If I remember correctly, `scp` fails if the remote server echoes a message like this during the login stage.

I prefer failing silently, i.e. nothing happens if you don't have plenv.